### DISCUSSION TITLE
Value combining "tree" for nested checkboxes

### DISCUSSION THREAD

**Discussion by tb-lbi** [directus/directus#14725] [Feature Requests]
It would be very useful to have a value combining option for nested checkboxes with the following behavior:
 - if any descendent of a checkbox is checked, then that checkbox's value is stored as well
 - the user may check a box without checking any of its children

**Comment by rijkvanzanten** [comment]
I believe both of those are supported by the current Value Combining option 🙂 

<img width="666" alt="CleanShot 2022-07-28 at 15 22 01@2x" src="https://user-images.githubusercontent.com/9141017/181620439-02e14927-87ff-416b-b6f9-c089255630d5.png">

- `Indeterminate` is "if any descendent of a checkbox is checked, then that checkbox's value is stored as well"
- `Exclusive` should handle "the user may check a box without checking any of its children"

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
